DOCS/compile-windows.md

```diff
@@ -145,7 +145,12 @@ they will be automatically downloaded and built by Meson.
    - From the installer, select the necessary components:
       - Clang compiler for Windows
       - Windows SDK
```

Comment:
I would put CMake here
```
- C++ CMake tools for Windows
```

Reply:
The reason I did this way was because someone might prefer their own way to install it (e.g. `scoop install cmake`). But now I believe people are smart enough to figure out this by themselves.